syslog-ng-ctl/syslog-ng-ctl.c

```diff
@@ -61,6 +61,34 @@ slng_reload(int argc, char *argv[], const gchar *mode, GOptionContext *ctx)
   return dispatch_command("RELOAD");
 }
 
+static gboolean config_options_start = FALSE;
+static gboolean config_options_stop = FALSE;
+
+static GOptionEntry internal_options[] =
+{
+  { "start", 's', 0, G_OPTION_ARG_NONE, &config_options_start, "start live collection of internal logs", NULL },
+  { "stop", 'x', 0, G_OPTION_ARG_NONE, &config_options_stop, "stop live collection of internal logs", NULL },
+  { NULL,           0,   0, G_OPTION_ARG_NONE, NULL,                         NULL,           NULL }
+};
+
+static gint
+slng_internal(int argc, char *argv[], const gchar *mode, GOptionContext *ctx)
+{
+  GString *cmd = g_string_new("");
+
```

Comment:
If neither start or stop are provided, I think we should write an error. Now the cli exits silently. Well the error code is not zero, so not entirely silently. Still, we could write something out.

Reply:
@furiel : Okay, I have added the error message for cases where neither 'start' nor 'stop' is provided.